packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -45,6 +48,11 @@ const propTypes = {
    * Whether or not the result has a value for a specific datetime.
    */
   hasResultNoData: PropTypes.bool,
+  /**
+   * /**
```

Comment:
nit: remove this line

Reply:
https://github.com/cerner/terra-clinical/pull/868/commits/ba6cb16362db09a115cd6bb88d36f7853bfabc10#diff-42afe5315e2b63b43f755f1ec567ef32b1ad903109d718db4e7526f825680f6aR51